networking/external_dns_operator/nw-control-dns-records-public-hosted-zone-infoblox.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-creating-dns-records-on-infoblox.adoc
+
+:_content-type: PROCEDURE
+[id="nw-control-dns-records-public-hosted-zone-infoblox_{context}"]
+= Creating DNS records on an public hosted zone for Infoblox by using Red Hat External DNS Operator
+
+You can create DNS records on a public hosted zone for Infoblox by using the Red Hat External DNS Operator.
+
+.Procedure
+
+. Optional: Add CA as trusted to ExternalDNS Operator for Infoblox if Infoblox uses a self signed certificate:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator create configmap trusted-ca-infoblox --from-file=ca-bundle.crt=/path/to/pem/encoded/infoblox/ca
+----
++
+[source,terminal]
+----
+$ oc -n external-dns-operator patch subscription external-dns-operator --type='json' -p='[{"op": "add", "path": "/spec/config", "value":{"env":[{"name":"TRUSTED_CA_CONFIGMAP_NAME","value":"trusted-ca-infoblox"}]}}]'
+----
+
+. Create a `secret` with Infoblox credentials:
++
+----
+$ oc -n external-dns-operator create secret generic infoblox-credentials --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_USERNAME=${INFOBLOX_USERNAME} --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_PASSWORD=${INFOBLOX_PASSWORD}
+----
+
+. Get the routes to check your cluster domain:
++
+[source,terminal]
+----
+$ oc get routes --all-namespaces | grep console
+----
++
+.Example Output
+----
+openshift-console          console             console-openshift-console.apps.aws.devcluster.openshift.com                       console             https   reencrypt/Redirect     None
+openshift-console          downloads           downloads-openshift-console.apps.aws.devcluster.openshift.com                     downloads           http    edge/Redirect          None
+----
+
+.From the UI
+
+. Add an authoritative DNS zone for your cluster domain to Infoblox using the {product-title} web console
```

Comment:
`{product-title}` won't work here I think, as the console which needs to be used here is Infoblox' one, not Red Hat OpenShift.

Reply:
Ack'ed